Update the readme (#3703)
stripe/stripe-android: README.md

<!--te-->

## Features

**Simplified Security**: Use the SDK to collect credit card numbers and remain [PCI compliant](https://stripe.com/docs/security#pci-dss-guidelines). This means sensitive data is sent directly to Stripe instead of passing through your server. For more information, see our [Integration Security Guide](https://stripe.com/docs/security).

**SCA-Ready**: The SDK automatically performs native [3D Secure authentication](https://stripe.com/docs/payments/3d-secure) to comply with [Strong Customer Authentication](https://stripe.com/docs/strong-customer-authentication) regulation in Europe.

**Native UI**: We provide native screens and elements to collect payment. For example, [PaymentSheet](https://stripe.com/payments/accept-a-payment?platform=android) is a prebuilt UI that combines all the steps required to pay - collecting payment details, and confirming the payment  - into a single sheet that displays on top of your app.

<img src="https://raw.githubusercontent.com/stripe/stripe-android/michelleb/mc-update-readme/assets/payment_sheet_complete.png"/>

**Google Pay**: Stripe is fully compatible with [Google Pay](https://stripe.com/docs/google-pay).

> **Stripe API**: We provide [low-level APIs](https://stripe.dev/stripe-android/stripe/com.stripe.android/-stripe/index.html) that correspond to objects and methods in the Stripe API. You can build your own entirely custom UI on top of this layer.

Author: ```suggestion
**Stripe API**: We provide [low-level APIs](https://stripe.dev/stripe-android/stripe/com.stripe.android/-stripe/index.html) that correspond to objects and methods in the Stripe API. You can build a custom UI on top of this layer.
```

## Features

**Native UI**: We provide native screens and elements to collect payment. For example, [PaymentSheet](https://stripe.com/payments/accept-a-payment?platform=android) is a view that combines all the steps required to pay - collecting payment details, billing details, and confirming the payment  - into a single sheet that displays on top of your app.

> <img width="270" height="555" src="https://raw.githubusercontent.com/stripe/stripe-android/master/assets/card_input.gif"/>

Reviewer: Do we have a new asset to use?

Author: yep, updated

**Simplified Security**: Use the SDK to collect credit card numbers and remain [PCI compliant](https://stripe.com/docs/security#pci-dss-guidelines). This means sensitive data is sent directly to Stripe instead of passing through your server. For more information, see our [Integration Security Guide](https://stripe.com/docs/security).

**SCA-Ready**: The SDK automatically performs native [3D Secure authentication](https://stripe.com/docs/payments/3d-secure) to comply with [Strong Customer Authentication](https://stripe.com/docs/strong-customer-authentication) regulation in Europe.

> **Native UI**: We provide native screens and elements to collect payment. For example, [PaymentSheet](https://stripe.com/payments/accept-a-payment?platform=android) is a prebuilt UI that combines all the steps required to pay - collecting payment details, billing details, and confirming the payment  - into a single sheet that displays on top of your app.

Reviewer: Looks like we sometimes use "prebuilt" and sometimes "pre-built"

Reviewer 2: Guidance is to use "prebuilt"

<img src="https://raw.githubusercontent.com/stripe/stripe-android/michelleb/mc-update-readme/assets/payment_sheet_complete.png"/>

**Google Pay**: Stripe is fully compatible with [Google Pay](https://stripe.com/docs/google-pay).

**Stripe API**: We provide [low-level APIs](https://stripe.dev/stripe-android/stripe/com.stripe.android/-stripe/index.html) that correspond to objects and methods in the Stripe API. You can build your own entirely custom UI on top of this layer.

## Getting Started

Get started with our [📚 integration guides](https://stripe.com/docs/payments) and [example projects](#examples), or [📘 browse the SDK reference](https://stripe.dev/stripe-android/).

The `PaymentSheet` class is the entry-point to the pre-built UI. It must be instantiated with a [Stripe publishable key](https://stripe.com/docs/keys) and application context.

When testing, you can use a test publishable key. Remember to replace the test key with your live key in production. You can view your API keys in the [Stripe Dashboard](https://dashboard.stripe.com/apikeys).

```java
PaymentConfiguration.init(getApplicationContext(), "pk_test_...");

> ```

Reviewer: Let's switch to Kotlin

## Examples

> - The [example project](https://github.com/stripe/stripe-android/tree/master/example) demonstrates using our prebuilt ui (single-step and multi-step), card element, and API bindings, including how to create Payment Intents, and Payment Methods; how to use the Stripe class's synchronous and asynchronous methods; and how to use the CardInputWidget.

Reviewer: This might be easier to read as a list of features.

- [Accept a card payment](https://github.com/stripe-samples/accept-a-card-payment) (PaymentIntents API)

> - [Save a card without payment](https://github.com/stripe-samples/mobile-saving-card-without-payment) (SetupIntents API)

Reviewer 2: I'd delete these too.

**Simplified Security**: Use the SDK to collect credit card numbers and remain [PCI compliant](https://stripe.com/docs/security#pci-dss-guidelines). This means sensitive data is sent directly to Stripe instead of passing through your server. For more information, see our [Integration Security Guide](https://stripe.com/docs/security).

**SCA-Ready**: The SDK automatically performs native [3D Secure authentication](https://stripe.com/docs/payments/3d-secure) to comply with [Strong Customer Authentication](https://stripe.com/docs/strong-customer-authentication) regulation in Europe.

**Native UI**: We provide native screens and elements to collect payment. For example, [PaymentSheet](https://stripe.com/payments/accept-a-payment?platform=android) is a prebuilt UI that combines all the steps required to pay - collecting payment details, billing details, and confirming the payment  - into a single sheet that displays on top of your app.

<img src="https://raw.githubusercontent.com/stripe/stripe-android/michelleb/mc-update-readme/assets/payment_sheet_complete.png"/>

**Google Pay**: Stripe is fully compatible with [Google Pay](https://stripe.com/docs/google-pay).

> **Stripe API**: We provide [low-level APIs](https://stripe.dev/stripe-android/stripe/com.stripe.android/-stripe/index.html) that correspond to objects and methods in the Stripe API. You can build your own entirely custom UI on top of this layer.

Reviewer: > You can build your own entirely custom UI on top of this layer.

How about "You can build a custom UI on top of this layer."

## Getting Started

Get started with our [📚 integration guides](https://stripe.com/docs/payments) and [example projects](#examples), or [📘 browse the SDK reference](https://stripe.dev/stripe-android/).

> The `PaymentSheet` class is the entry-point to the pre-built UI. It must be instantiated with a [Stripe publishable key](https://stripe.com/docs/keys) and application context.

Reviewer: Can you clarify what it means that it needs to be instantiated with a publishable key and application context? Do you mean that `PaymentConfiguration` has to be instantiated first?

Reviewer: Also `PaymentConfiguration` will always grab `applicationContext` so it can really be instantiated with any context.

Author: I am removing this to keep it consistent with ios.  There is more than just this that needs to be done for PaymentSheet (see guide)

```

## Examples

> - The [example project](https://github.com/stripe/stripe-android/tree/master/example) demonstrates using our prebuild ui (single-step), prebuild UI (multi-step), card element, and API bindings, including how to create Payment Intents, and Payment Methods; how to use the Stripe class's synchronous and asynchronous methods; and how to use the CardInputWidget.

Reviewer: Is it prebuild, prebuilt, pre-build, or pre-built? Not sure what is preferred.

Author: This will match yuki's documentation.

Reviewer: Sg
